rpcserver.go

```diff
@@ -494,64 +494,46 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 
 	var seedlingMeta *proof.MetaReveal
 
-	// If a custom decimal display is set, the meta type must also be set to
-	// JSON.
+	// If a custom decimal display is set, we require the AssetMeta to be
+	// set. That means the user has to at least specify the meta type.
```

Comment:
What about assuming it's just the opaque type if the dec display is set? Given that by default (opaque asset type, not JSON), we won't encode this in the asset meta.

Reply:
Hmm, yeah, I guess we could do that now. But that would mean that both the meta data blob and type would be empty/unset and the meta hash would just be TLV record of the decimal display?